src/content/docs/en/guides/integrations-guide/deno.mdx

```diff
@@ -1,188 +1,13 @@
 ---
-# NOTE: This file is auto-generated from 'scripts/generate-integration-pages.ts'
-#       and pulls content directly from the package’s README.
-#       DO NOT MAKE EDITS TO THIS FILE DIRECTLY, THEY WILL BE OVERWRITTEN!
-#       For corrections, please edit the package README at
-#       https://github.com/withastro/astro/tree/main/packages/integrations/deno/
-#
-# TRANSLATORS: please remove this note and the <DontEditWarning/> component.
-
 type: integration
 title: '@astrojs/deno'
-description: Learn how to use the @astrojs/deno SSR adapter to deploy your Astro project.
-githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/deno/'
+description: The Deno Astro adapter
+githubURL: https://github.com/withastro/docs/tree/main/src/content/docs/en/guides/integrations-guide/deno.mdx
 hasREADME: true
```

Comment:
@delucis Should we set this to false if we are not pulling in a README but rather setting our own page manually?
```suggestion
hasREADME: false
```

Reply:
Hmm. We might need to tweak some site code here potentially.

We use `githubURL` to set the values for the **GitHub** and **Changelog** links here: 
<img width="448" alt="Header for the @astrojs/deno page" src="https://github.com/withastro/docs/assets/357379/7f7bf50e-b83d-46e2-b95b-9c5d3822d28c">

So if we’re maintaining how those behave, we should probably make `githubURL` the Deno repo and keep `hasREADME: true` (`true` distinguishes integrations from the config reference that comes from the types file). BUT that will mean the edit link is wrong…

Alternatively, we remove `type: integration` so that this page looks like a standard page in docs without the integration-specific header, and remove `githubURL` and `hasREADME`. This would have the side effect of removing the Deno logo from the integrations nav component (and the integrations nav from the footer of this page — although we could easily add it back manually).

I was arguing on Discord for aiming to keep this around in the integrations nav for a bit, but if that requires some custom code just for this one page… maybe that’s not worth the effort 😅

Reply:
OK, so yeah, it seems like removing `type: integration` is the path of least resistance! :sweat_smile: 

I propose we do that, it removes Deno from the list, and then the edit this page link should be correct, right? I'll ask you to propose the change that does this properly, and I'll handle adding migration instructions to the text.